src/Core/LanguageService.fs

```diff
@@ -599,7 +603,37 @@ Consider:
         let clientOpts =
             let opts = createEmpty<Client.LanguageClientOptions>
 
+            let mutable initFails = 0
+
+            let initializationFailureHandler (error:  U3<ResponseError,Exception,obj option>) =
+                if initFails < 5 then
+                    logger.Error($"Initialization failed: %%", error)
```

Comment:
Is %% some kind of special format string?

Reply:
According to https://github.com/ionide/ionide-vscode-fsharp/blob/9e63abe1301a1dc18b3bc0ac839386ee0e879dde/src/Core/Logging.fs#L151-L155 yes